lineman/app/models/discussion_records_interface.coffee

```diff
@@ -3,13 +3,18 @@ angular.module('loomioApp').factory 'DiscussionRecordsInterface', (BaseRecordsIn
     model: DiscussionModel
 
     fetchByGroup: (options = {}) ->
-      @restfulClient.getCollection
+      @fetch
```

Comment:
I never liked that we were calling @restfulClient directly from the child classes, and turns out this is why: we want everything to go through fetch so behaviour can be as consistent as possible.